### DISCUSSION TITLE
Collapsible columns

### DISCUSSION THREAD

**Discussion by binaryfire** [mokhosh/filament-kanban#41] [Ideas]
Hi @mokhosh. Great package!

Would you consider adding the ability to collapse columns (and also set the default collapsed / expanded state of colums)? This is very common in project management tools. For example the first column may be "Ideas" and the last column might be "Completed" - you'd normally want both of those collapsed so they're not taking up space.

Here's an example from GitLab:

Open (not collapsed):
![image](https://github.com/mokhosh/filament-kanban/assets/5361908/5783ab6a-1e38-4054-8734-6896a8bb63d1)

Closed (collapsed):
![image](https://github.com/mokhosh/filament-kanban/assets/5361908/a0d18c87-5204-4496-bb3c-4d4a8d825d04)

**Comment by mokhosh** [comment]
Nice idea. I'll try to see if I can do this, but I don't know when, as I'm currently very busy with work.

**Comment by willcastillo** [comment]
FWIW, I think a filter would be a better UI/UX to achieve this.

![image](https://github.com/mokhosh/filament-kanban/assets/1260357/9fa044e6-563f-4492-b209-750bf86f2dac)
